src/components/NavBar.module.css

```
@@ -1,105 +1,270 @@
-    top: 100%;
-    right: 0;
+    top: calc(100% + var(--spacing-sm));
+    left: var(--spacing-sm);
+    right: var(--spacing-sm);
     flex-direction: column;
-    width: 200px;
-    padding: 1rem;
-    transform: translateX(100%);
-    transition: transform 0.3s ease;
+    gap: 6px;
+    padding: 12px;
+    background: linear-gradient(145deg, 
+      rgba(106, 56, 23, 0.95) 0%, 
+      rgba(83, 46, 26, 0.98) 100%);
+    border: 1px solid rgba(215, 164, 125, 0.2);
+    border-radius: 20px;
+    box-shadow: 
+      0 20px 50px rgba(83, 46, 26, 0.4),
+      0 0 0 1px rgba(248, 234, 205, 0.05) inset;
+    opacity: 0;
+    visibility: hidden;
+    transform: translateY(-8px);
+    z-index: var(--navbar-menu-z-index);
+    transition: 
+      opacity 0.4s cubic-bezier(0.16, 1, 0.3, 1),
+      transform 0.4s cubic-bezier(0.16, 1, 0.3, 1),
+      visibility 0.4s;
+  }
+
+  .navList.open {
+    opacity: 1;
+    visibility: visible;
+    transform: translateY(0);
+  }
+
+  .navList li {
+    width: 100%;
+    opacity: 0;
+    transform: translateX(-12px);
+    transition: 
+      opacity 0.35s cubic-bezier(0.16, 1, 0.3, 1),
+      transform 0.35s cubic-bezier(0.16, 1, 0.3, 1);
   }
 
-  .navLinks.active {
+  .navList.open li {
+    opacity: 1;
     transform: translateX(0);
   }
 
-  .navLinks li {
-    margin: 0.25rem 0;
+  .navList.open li:nth-child(1) { transition-delay: 0.03s; }
+  .navList.open li:nth-child(2) { transition-delay: 0.06s; }
+  .navList.open li:nth-child(3) { transition-delay: 0.09s; }
+  .navList.open li:nth-child(4) { transition-delay: 0.12s; }
+  .navList.open li:nth-child(5) { transition-delay: 0.15s; }
+
+  .navLink {
+    width: 100%;
+    padding: 14px 16px;
+    justify-content: flex-start;
+    border-radius: 12px;
+    background: transparent;
+    border: none;
+    color: rgba(248, 234, 205, 0.85);
+    font-size: 0.95rem;
   }
-}
 
-.externalLink {
-  margin-left: 1.25em;
-}
+  .navLink svg:first-child {
+    opacity: 0.6;
+    transition: opacity 0.25s ease, color 0.25s ease;
+  }
+
+  .navLink:hover {
+    background: rgba(248, 234, 205, 0.1);
+    color: #F8EACD;
+    transform: none;
+  }
+
+  .navLink:hover svg:first-child {
+    opacity: 1;
+    color: var(--color-secondary);
+  }
 
-.externalLinkIcon {
+  .navLink:active {
+    background: rgba(248, 234, 205, 0.15);
+    transform: scale(0.98);
+  }
+
+  .navLink.active {
+    background: linear-gradient(135deg, 
+      rgba(248, 234, 205, 0.25) 0%, 
+      rgba(215, 164, 125, 0.2) 100%);
+    color: #F8EACD;
+    font-weight: 600;
+    border-left: 3px solid #F8EACD;
+  }
+
+  .navLink.active svg:first-child {
+    opacity: 1;
+    color: #F8EACD;
+  }
+
+  /* Divider between internal and external links */
+  .navList li:nth-child(3)::after {
+    content: '';
+    display: block;
+    height: 1px;
+    background: linear-gradient(90deg, transparent, rgba(215, 164, 125, 0.25), transparent);
+    margin-top: 6px;
+  }
+}.externalLinkIcon {
```

Just for formatting consistency.

```suggestion
}

.externalLinkIcon {
```